jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/listeners/SessionSetupListener.java

```
@@ -39,10 +47,19 @@
      */
     @Override
     public void sessionCreated(HttpSessionEvent se) {
-        ServletContext servletContext = se.getSession().getServletContext();
-        ApplicationContext ctx = WebApplicationContextUtils.getRequiredWebApplicationContext(servletContext);
-        JCommuneProperty property = (JCommuneProperty) ctx.getBean("sessionTimeoutProperty");
-        int timeoutInSeconds = (int) TimeUnit.SECONDS.convert(property.intValue(), TimeUnit.MINUTES);
+        ServletContext servletContext;
+        ApplicationContext ctx;
+        JCommuneProperty property = null;
+        try {
+            servletContext = se.getSession().getServletContext();
+            ctx = WebApplicationContextUtils.getRequiredWebApplicationContext(servletContext);
+            property = (JCommuneProperty) ctx.getBean("sessionTimeoutProperty");
+            locationService = (LocationService) ctx.getBean("locationService");
+        } catch (Exception ex) {
+            logger.warn("Bean instantiation error: " + ex.getMessage());
```

Shouldn't we re-throw the error to make the app crash during startup?